src/dmd/backend/go.c

```diff
@@ -120,7 +120,7 @@ int go_flag(char *cp)
             case GLspace:
                 go.mfoptim |= MFtime;      /* inverse of -time     */
                 break;
-            case -1:                    /* not in flagtab[]     */
+            case 255:                    /* not in flagtab[]     */
```

Comment:
Better to change the declaration of `flag` from `unsigned` to `int`.

Reply:
Cool! Done